data/ExperimentDesignPresets.ts

```diff
@@ -16,22 +16,28 @@ interface Preset<T> {
   preset: RecursivePartial<T>;
 }
 
-const presets: { [id: string]: Preset<ExperimentDesign> } = {
+const presets: { [id: string]: Preset<ExperimentRecipe> } = {
   empty_aa: {
     name: "A/A Experiment",
     description:
       "A design for diagnostic testing of targeting or enrollment. Fixed to 1% of the population.",
     preset: {
-      proposedDuration: 28,
-      proposedEnrollment: 7,
-      branches: [
-        { slug: "control", ratio: 1, value: null },
-        { slug: "treatment", ratio: 1, value: null },
-      ],
-      bucketConfig: {
-        randomizationUnit: "normandy_id",
-        count: 100,
-        total: 10000,
+      filter_expression:
+        "env.version|versionCompare('{minFirefoxVersion}') >= 0 && {audience_filter_expression}",
```

Comment:
Audiences don't have a `filter_expression` so do we need this @k88hudson ?

https://github.com/mozilla/nimbus-shared/blob/main/data/Audiences.ts#L4-L11

Reply:
Oh I just saw your comment.  Well do we want to

1) Omit it here since it isn't being used yet
2) Include it here and add a dummy value to audiences like `true`
3) Something else?

Reply:
Audiences don't need to use `filter_expression` for anything at the moment, but it's defined as optional on the type https://github.com/mozilla/nimbus-shared/blob/main/types/targeting.ts#L5

Reply:
Tests in this repo or tests in the experimenter repo?

Reply:
Also do you mean verifying the template variable names ie `audienceFilterExpression` or the JEXL syntax literals like `versionCompare`?